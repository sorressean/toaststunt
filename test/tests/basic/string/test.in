length("foo")
strsub("foobarbaz", "bar", "quxx")
index("foobar", "bar")
index("foobar", "baz")
index("fooBar", "bar")
index("fooBar", "baz")
index("fooBar", "bar", 1)
rindex("bazbarbazfoo", "baz")
rindex("bazbarbazfoo", "quxx")
rindex("bazbarBazfoo", "baz")
rindex("bazbarBazfoo", "baz", 1)
strcmp("abc", "abd") < 0
strcmp("abd", "abc") > 0
strcmp("abc", "abc")
decode_binary("foo")
decode_binary("foo~0D~0A")
decode_binary("foo~0Abar~0A")
decode_binary("foo~0D~0A", 1)
encode_binary("~foo")
encode_binary({"foo", 10}, {"bar", 13})
encode_binary("foo", 10, "bar", 13)
crypt("foobar", "SA")
