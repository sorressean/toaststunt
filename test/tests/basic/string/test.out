3
"fooquxxbaz"
4
0
4
0
0
7
0
7
1
1
1
0
{"foo"}
{"foo", 13, 10}
{"foo", 10, "bar", 10}
{102, 111, 111, 13, 10}
"~7Efoo"
"foo~0Abar~0D"
"foo~0Abar~0D"
"SAEmC5UwrAl2A"
